sdk/appcenter/src/test/java/com/microsoft/appcenter/channel/OneCollectorChannelListenerTest.java

```diff
@@ -157,15 +165,16 @@ public void enqueueConvertedLogs() {
         listener.onGloballyEnabled(true);
 
         /* Mock a 4rd log in first group to check reset. */
-        CommonSchemaLog log4 = mock(CommonSchemaLog.class);
+        CommonSchemaLog log4 = new MockCommonSchemaLog();
+        log4.setIKey("t2");
         Extensions ext4 = new Extensions();
         ext4.setSdk(new SdkExtension());
-        when(log4.getExt()).thenReturn(ext4);
-        when(log4.getIKey()).thenReturn("t2");
+        log4.setExt(ext4);
         when(logSerializer.toCommonSchemaLog(any(Log.class))).thenReturn(Collections.singletonList(log4));
-        listener.onPreparedLog(originalLog, TEST_GROUP, DEFAULT_FLAGS);
+        listener.onPreparedLog(originalLog, TEST_GROUP, PERSISTENCE_NORMAL);
 
-        /* Verify reset of epoch/seq. */
+        /* Verify flags and reset of epoch/seq. */
+        assertEquals(new Long(PERSISTENCE_NORMAL), log4.getFlags());
         assertEquals((Long) 1L, log4.getExt().getSdk().getSeq());
```

Comment:
Let's use only one way for casting numbers in asserts like this. For example
```suggestion
         assertEquals(Long.valueOf(1), log4.getExt().getSdk().getSeq());
```

Reply:
Ok.